app/api/utils.py

```diff
@@ -320,11 +321,20 @@ def parse(self, file):
         reader = csv.reader(file)
         columns = next(reader)
         data = []
+        if len(columns) == 1 and columns[0] != 'text':
+            if len(data) >= settings.IMPORT_BATCH_SIZE:
+                yield data
+                data = []
+            data.append({'text': columns[0]})
```

Comment:
Good catch. This is for headerless CSVs, correct?

Reply:
Yeah it's to handle single column csv files (just the text) with or without a header